src/lib/services/instance-stats-service.ts

```diff
@@ -111,7 +119,37 @@ export class InstanceStatsService {
         this.logger = getLogger('services/stats-service.js');
     }
 
-    async getToggleCount(): Promise<number> {
+    async start(): Promise<void> {
+        if (!this.snapshotRefresher) {
+            this.refreshStatsSnapshot();
```

Comment:
This doesn't get awaited, is there a reason for that?

Reply:
My idea is that by not waiting on it, it'll run in the background. Anyhow this `start` method will go away as I plan to migrate to the newly scheduler service introduced a few days ago